libra_validation/fssh_based_2_state_models/recipes.py

```diff
@@ -157,3 +160,347 @@ def make_all_sets():
 
     return all_sets
 
+def set_recipe_v2(dyn_general, recipe, name=""):
+    """
+    This function sets a recipe for which the calculations should be run.
+    Args:
+        dyn_general (dictionary): This dictionary contains necessary variables to perform nonadiabatic dynamics.
+                                 It will be updated and returned. 
+        recipe (integer list): This list contains 10 values
+                               1st: Model (A)
+                                   There are currently 4 options for the models
+                               2nd: Initial conditions (B)

```

Comment:
initial condition specification should also include information about the electronic state initialization